model/src/channel/message/mod.rs

```diff
@@ -101,23 +115,44 @@ pub struct Message {
     ///
     /// [Message Content Intent]: crate::gateway::Intents::MESSAGE_CONTENT
     pub embeds: Vec<Embed>,
+    /// Enabled [`MessageFlags`].
     #[serde(skip_serializing_if = "Option::is_none")]
     pub flags: Option<MessageFlags>,
+    /// ID of the [`Guild`] the message was sent in.
+    ///
+    /// [`Guild`]: crate::guild::Guild
```

Comment:
I tried to add these sort of interlinks where applicable, should they be expanded on or removed entirely?

Reply:
i don't know, maybe? i'd like to have some other input on this